### ISSUE TITLE
Radar works without power

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#848]
```
Game version: 0.95.174


Operating System: Windows 8.1 64bit


Java runtime version: 8


Installed using the Launcher? yes


Game language (en, hu, de): en


Report: 

Radar station work without power. Destroying them removes the planets' radar 
range, deactivating them does not.

PS: Long time no see :)




```

Original issue reported on code.google.com by `norbert....@gmail.com` on 20 Apr 2014 at 6:16

**Comment by GoogleCodeExporter** [comment]
```
Thanks.
```

Original comment by `akarn...@gmail.com` on 29 Apr 2014 at 6:48
- Changed state: **SVN**
- Added labels: **Component-Logic**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.95.175
```

Original comment by `akarn...@gmail.com` on 26 May 2014 at 1:35
- Changed state: **Fixed**
